app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -194,7 +194,10 @@ public boolean onPreferenceTreeClick(PreferenceScreen preferenceScreen, Preferen
             showSettingsFragment(SettingsScreen.SEARCH_ENGINES);
             TelemetryWrapper.openSearchSettingsEvent();
         } else if (preference.getKey().equals(resources.getString(R.string.pref_key_manual_add_search_engine))) {
-            showSettingsFragment(SettingsScreen.ADD_SEARCH);
+            getFragmentManager().beginTransaction()
```

Comment:
This statement is equivalent to the one you removed but less high-level – did you have a particular reason for changing the code? I find the high-level `showSettingsFragment(SettingsScreen.ADD_SEARCH)` to be clearer and it lets our SettingsScreen enum tell us all the different search screens we can show (if it's kept up to date).